examples/benchmarks.py

```diff
@@ -24,7 +24,14 @@
 from time import time
 from typing import List
 
-from transformers import AutoConfig, AutoTokenizer, is_tf_available, is_torch_available
+from transformers import (
+    AutoConfig,
+    AutoTokenizer,
+    is_tf_available,
+    is_torch_available,
+    start_memory_tracing,
+    stop_memory_tracing,
+)
```

Comment:
I would love this in `src/transformers/utils_benchmarks.py` so that unit tests can use it!

Reply:
(this was dumb, it already works)